ddtrace/internal/_encoding.pyx

```diff
@@ -101,7 +101,7 @@ cdef inline int pack_number(msgpack_packer *pk, object n):
     raise TypeError("Unhandled numeric type: %r" % type(n))
 
 
-cdef inline int pack_text(msgpack_packer *pk, object text):
+cdef inline int pack_text(msgpack_packer *pk, object text) except -1:
```

Comment:
We want these to be `except? -1`. Which tells the caller if it receives `-1` there _might_ be a Python exception on the stack, but because we also return from `msgpack_*` functions which return `-1` but do not put a Python exception on the stack, then we want the caller to be aware.